cmd/common-methods.go

```diff
@@ -24,9 +24,11 @@ import (
 	"regexp"
 	"strings"
 
+	"golang.org/x/net/http/httpguts"
+	"gopkg.in/h2non/filetype.v1"
```

Comment:
We could also have used `net/http` package `DetectContentType()` function instead of `filetype` package.

Reply:
DetectContentType gives wrong values on certain files @ebozduman